Model/Api/Request.php

```diff
@@ -28,30 +37,56 @@ public function __construct(
         if (!empty($params['requestType'])) {
             $this->requestType = $params['requestType'];
         }
+
+        if (class_exists(\Laminas\Http\Client::class)) {
+            $this->client = new \Laminas\Http\Client();
+        } elseif (class_exists(\GuzzleHttp\Client::class)) {
+            $this->client = new \GuzzleHttp\Client();
+        } elseif (class_exists(\Zend\Http\Client::class)) {
+            $this->client = new \Zend\Http\Client();
+        } else {
+            throw new Exception('No HTTP client library available.');
+        }
     }
 
     public function send($data)
     {
         $apiKey = $this->config->apiKey();
         $secret = $this->config->secret();
         $apiUrl = $this->env->apiUrl();
-
-        $client = new \Laminas\Http\Client();
-
-        $client->setUri("$apiUrl/{$this->path}?api_key=$apiKey");
-        $client->setOptions(['timeout' => $this->timeout]);
-        $client->setMethod($this->requestType);
-        $client->setRawBody(json_encode($data));
-        $client->setHeaders([
+        $headers = [
             'X-Authorization' => strval($secret),
-            'Content-Type'    => 'application/json'
-        ]);
+            'Content-Type' => 'application/json',
+        ];
+        $url = "$apiUrl/{$this->path}?api_key=$apiKey";
+
+        if ($this->client instanceof \Laminas\Http\Client) {
+            $this->client->setUri($url);
+            $this->client->setOptions(['timeout' => $this->timeout]);
+            $this->client->setMethod($this->requestType);
+            $this->client->setRawBody(json_encode($data));
+            $this->client->setHeaders($headers);
+            $this->response = $this->client->send();
+            $result = $this->response->isSuccess() ? json_decode($this->response->getBody(), true) : false;
+        } elseif ($this->client instanceof \GuzzleHttp\Client) {
 
-        $this->response = $client->send();
+            $options = [
+                'headers' => $headers,
+                'json' => $data,
+                'timeout' => $this->timeout,
+            ];
+            $response = $this->client->request($this->requestType, $url, $options);
+            $this->response = $response;
+            $result = $response->getStatusCode() === 200 ? json_decode($response->getBody(), true) : false;
 
-        $result = $this->response->isSuccess();
-        if ($result) {
-            $result = json_decode($this->response->getBody(), true);
+        } elseif ($this->client instanceof \Zend\Http\Client) {
+            $this->client->setUri($url);
+            $this->client->setOptions(['timeout' => $this->timeout]);
+            $this->client->setMethod($this->requestType);
+            $this->client->setRawBody(json_encode($data));
+            $this->client->setHeaders($headers);
+            $this->response = $this->client->send();
+            $result = $this->response->isSuccess() ? json_decode($this->response->getBody(), true) : false;
```

Comment:
`\Laminas\Http\Client` and `\Zend\Http\Client` seems to be identical. We can simplify and dry code with
```
if ($this->client instanceof \GuzzleHttp\Client) {
    $options = [
        'headers' => $headers,
        'json'    => $data,
        'timeout' => $this->timeout,
     ];
     $response = $this->client->request($this->requestType, $url, $options);
     $this->response = $response;
     $result = $response->getStatusCode() === 200 ? json_decode($response->getBody(), true) : false;
} else {
    $this->client->setUri($url);
    $this->client->setOptions(['timeout' => $this->timeout]);
    $this->client->setMethod($this->requestType);
    $this->client->setRawBody(json_encode($data));
    $this->client->setHeaders($headers);
    $this->response = $this->client->send();
  $result = $this->response->isSuccess() ? json_decode($this->response->getBody(), true) : false;
}
```